match/main/management/commands/export_markdown.py

```diff
@@ -14,23 +14,23 @@ class Command(BaseCommand):
 
     SHORTCODES = {
         'counselingsession':
-            '{{< interactives url="dentalvisit" width="900px"' +
-            ' height="800px" >}}',
+            '\n\n{{< interactives url="dentalvisit" minsize="sm"' +
```

Comment:
Zarina: I needed two line breaks so that Hugo can make it the block a block, instead of some weird inline formatting.